packages/components/molecules/f-mega-modal/vue.config.js

```diff
@@ -5,6 +5,19 @@ const sassOptions = require('../../../../config/sassOptions')(rootDir);
 
 // vue.config.js
 module.exports = {
+    configureWebpack: {
```

Comment:
Are you able to explain the intention of this change? Is it doing something specifically to help with bundle sizes?

Reply:
Correct. It splits the anything within `node_modules` into its own bundle.

Currently CircleCI has issues building f-checkout / f-searchbox due to memory issues on the free plan, so have to focus on bundle optimisation to unblock CI until we get approval for a paid plan 👍🏼.


There's defiantly another conversation to be had on bundle optimisation I think. From mine and @matthewhardern's initial investigation, there's a lot more we could be doing around splitting / compression.